environment.yml

```diff
@@ -81,10 +81,11 @@ dependencies:
   - conda-forge::gprof2dot
   - conda-forge::numdifftools
   - conda-forge::quantities
+  - conda-forge::lpsolve55
+  - conda-forge::ringdecomposerlib-python
+  - conda-forge::muq
 
 # packages we maintain
-  - rmg::lpsolve55
-  - rmg::muq2
   - rmg::pydas >=1.0.3
   - rmg::pydqed >=1.0.3
   - rmg::pyrdl
```

Comment:
Was the intention to remove `rmg::py_rdl` ?

Reply:
Oops thanks for noticing, resolved a previous merge conflict incorrectly